src/main/resources/db/migration/V1_41__DROP_CMS_EXPORT_VIEW.sql

```diff
@@ -0,0 +1 @@
+DROP VIEW app.cms_export_view;
```

Comment:
```suggestion
DROP MATERIALIZED VIEW app.cms_export_view;
```

does this need to be this?